test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -1692,7 +1824,13 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 		})
 
 		AfterEach(func() {
-			//Revert the profile
+			profile, err = profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
+			Expect(err).ToNot(HaveOccurred())
+
+			if profile.ResourceVersion == initialProfile.ResourceVersion {
```

Comment:
at this point you will have for sure initialProfile = profile.DeepCopy()

Reply:
Is probably clearer and more explicit to add a explicit skip flag in the `It` bodies?

Reply:
Putting it inside the `It` has 2 disadvantages:
1. When new `It`s are added it's easy to forget to add this check
2. Skipping inside the `It` doesn't prevent the `AfterEach()` from running.

NOTE: we changed the logic to examine only the `Spec` since the `Status` filed might change and update the `ResourceVersion`

Reply:
Good points. What bothers me at glance is relying on ResourceVersion. I guess we can iterate a bit over it, and I can review again a bit deeper anyway to doublecheck this is robust enough.

Reply:
> What bothers me at glance is relying on ResourceVersion

Read my NOTE. We're not anymore